common/rdm/RDMAPI.cpp

```diff
@@ -4413,7 +4413,8 @@ void RDMAPI::_HandleGetDimmerMaximumLevel(
     if (data.size() != DATA_SIZE) {
       SetIncorrectPDL(&response_status, data.size(), DATA_SIZE);
     } else {
-      maximum_level = data[0];
+      const uint16_t *ptr = reinterpret_cast<const uint16_t*>(data.data());
```

Comment:
Good catch, sorry we both missed it first time around. You can simply this slightly, if you look at how we do _HandleGetDMXAddress you don't need the ptr variable.

Reply:
Sounds good, I will tweak that today.